localstack/services/awslambda/lambda_executors.py

```diff
@@ -645,7 +647,7 @@ def run_lambda_executor(self, lambda_function: LambdaFunction, inv_context: Invo
             result = to_str(result).strip()
         except Exception:
             pass
-        log_output = to_str(log_output).strip()
+
         # Note: The user's code may have been logging to stderr, in which case the logs
         # will be part of the "result" variable here. Hence, make sure that we extract
         # only the *last* line of "result" and consider anything above that as log output.
```

Comment:
I'd like to remove this code very much :) 

There is a comment on the same code, questioning the utility or if this code is even necessary. I tried to reproduce
it, by logging to `stderr` inside the lambda and this code was never hit.

remove it?

Reply:
Agreed that the handling of stdout/stderr is a bit clunky here, partly related to the contract defined by LambCI: 

> The default is single execution mode, which outputs all logging to stderr and the result of the handler to stdout.

https://github.com/lambci/docker-lambda#running-lambda-functions

Given that we're in the process of rewriting the Lambda implementation based on AWS base images, this code will soon become obsolete 👍  - so not sure how much value there is in updating it at this stage..

Reply:
Sure, that's what I did and tested, result goes to stdout. But even forcing some logging to stderr couldn't make the result appear in the `stderr` as the comment states.

Reply:
I am not sure the correct line here is marked (casting the log output to str should be done, docker client will return bytes here), but the problem would be that the stdout (result here) contains logs as well as the result (but result will always be the last line), not that the actual result got printed to stderr (which should never be the case).

Reply:
> I am not sure the correct line here is marked (casting the log output to str should be done, docker client will return bytes here),

Notice that the `to_str` is done inside `OutputLog`. 

- result (stdout) contains logs from the user code inside 
- log_output (stderr) contains the lambda logs - but I tried to print "user" logs into stderr, and was never able to do it, so I think the TODO

Reply:
Ah, now I get it. The thing is, we depend there on the behavior of lambci, which has to provide the output this way. We cannot however assert what versions of images might be present on some peoples PC.
I would totally be in favor investigating more if it is absolutely certain this will not happen and remove it, but since we are reworking lambda anyway, I do not think its worth the hassle of maybe breaking something, even though it might not do so.

Reply:
On digging more into this, I came to the conclusion it would be pretty safe to remove this. This should only be relevant for the local executor currently.

Reply:
ditto seems to be something that will be removed anyway.